config/boot.rb

```diff
@@ -1,3 +1,13 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+require 'rails/commands/server'
```

Comment:
I know, but I was wondering how to change the port and I wanted to try. Its should write in the master branch